README.md

```diff
@@ -120,6 +121,108 @@ engine.listen(process.env.port || 8000);
 
 Note that this feature is currently in beta which means that it is close to production-ready but has not been run in production yet. We appreciate all efforts to try this out and provide feedback.
 
+
+## Live Mixing (BETA)
+
+This feature gives the possibility to mix in a true live stream in a Channel Engine powered linear channel (VOD2Live).
+
+![RFC Drawing of Live-Mixing Functionality](rfc/channel_engine_vod_with_live.png)
+
+A beta-version of live-mixing with high availability support is available in the Channel Engine. This allows you to use a new component which can let you break in to a scheduled VOD event or Live stream event at any speficied time on top of the usual vod-to-live content. 
+To enable live-mixing, create a class which implements the following interface.
+
+```
+class MyStreamSwitchManager {
+  getSchedule() -> [ { eventId, assetId, title, type, start_time, end_time, uri, duration } ]
+}
+```
+
+The class's `getSchedule()` function should return a list of event objects in the following format:
+
+```
+{
+  "eventId": {
+      "type": "string",
+      "description": "Generated ID of the event. If not specified a uuid-based eventId will be generated"
```

Comment:
@Nfrederiksen We currently do not generate any ID if it is not set by the user, right?

Reply:
correct, we do not

Reply:
I will remove that sentence